joomla/joomla-cms
# [4.3] Tags: Don't put NULL into trim()

Pull Request for Issue #41466.

### Summary of Changes
When on PHP 8.1 and editing an article, adding an unknown tag triggers an AJAX call which throws a notice.


### Testing Instructions
1. You have to be on PHP 8.1 or higher
2. Edit an article and click into the tags field
3. Open the browser console to see the ajax request
4. Type a random tag-name into the tag field
5. See the request in the console and check the request answer


### Actual result BEFORE applying this Pull Request
A notice is thrown and the JS actually throws an error, too.


### Expected result AFTER applying this Pull Request
No notice, everything is hunky dorry. 😉 


### Link to documentations
Please select:
